packages/a11y-base/src/delegate-focus-mixin.js

```diff
@@ -82,9 +81,15 @@ export const DelegateFocusMixin = dedupeMixin(
        * @protected
        * @override
        */
-      focus() {
+      focus(options) {
```

Comment:
Out of scope of this PR, just to mention it for completeness sack: flow also has s focus method which would need the overload and probably.. even tho I'm not sure where you wanna draw the line.. because some flow components implement Focusable and are not really your web-components (e.g. native html and so on..) so not sure if that would be the right interface..or your way of implementing it would also cover other elements 

https://github.com/vaadin/flow/blob/eda19b9ecd987b16cd499be503deb07c5457875e/flow-server/src/main/java/com/vaadin/flow/component/Focusable.java#L108

Reply:
Thanks for pointing out. Could you please create a separate issue in Flow? I think some users could also benefit from `preventScroll: true`.

Reply:
Created https://github.com/vaadin/flow/issues/22078